plugins/nid-team/scripts/pr-dashboard/list-merged-without-reviewer.py

```
@@ -0,0 +1,53 @@
+
+results = []
+for item in data["items"]:
+    c = item.get("content", {})
+    if c.get("type") != "PullRequest":
+        continue
+    if item.get("primary Reviewer", ""):
+        continue
+    repo = c.get("repository", "")
+    number = str(c.get("number", ""))
+    title = c.get("title", "")
+    r = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)